src/components/ReviewsContainer/ButtonComments.jsx

```diff
@@ -0,0 +1,10 @@
+
+const ButtonComments = () => {
+    return(
+        <div>
```

Comment:
Similarly here, maybe <section> would be better. Be careful not to overuse <div>